### ISSUE TITLE
Open Top crates

### ISSUE THREAD

**Issue by AndrewF74** [treeDiM/StackBuilder#152]
Are there any options for open-top crates?
![image](https://user-images.githubusercontent.com/60245283/77804484-3dcb2800-7045-11ea-87fa-e8258388dd47.png)
![image](https://user-images.githubusercontent.com/60245283/77804661-b8944300-7045-11ea-9fbc-7f0bced911a9.png)
![image](https://user-images.githubusercontent.com/60245283/77804768-f1341c80-7045-11ea-938a-3d59e17ba49b.png)

**Comment by fgasnier71** [comment]
No, these are not available yet.
Would you buy a 2-year Premium account to get these?

**Comment by AndrewF74** [comment]
I am already encouraging the company that I work for to go for a 2-year premium account the moment that I saw that you added bottles to the program.

For the totes alone? No, just thought that I would point out another direction to incorporate. I found a sub-par way to represent the image for my current project but thought that I would share the recommendation with you.

I seriously do like your program and I am trying to get this company on a 2-year subscription. Mainly, just for the bottles alone. By the way, with a premium account how much variation do we have with the bottles? I am assuming that there is no plan for textures on bottles at this time right?

Also, you might want to consider adding standard labeling to shipping boxes/cylinders that represent the labeling requirements of IATA/ICAO and IMDG for the transportation of Hazardous Materials. Again, just a friendly suggestion… 😊

Andrew D. Forsythe
Receiving Manager
andrewf@4life.com<mailto:andrewf@4life.com>
Phone:801.562.3729
Receiving Group: 801. 256.3007 or x 23007
Fax: 801.562.3635
Biomedical Research Laboratories LLC
4Life Research LLC
"Science, Service, & Success"

**Comment by fgasnier71** [comment]
Adding texture/stickers to cylinders/bottles is not easy as revolution solid are drawn as a succession of small quadrilaterals and it would require cutting the image in small parts for drawn on each quadrilaterals.
We are not relying on a real 3D library (OpenGL or DirectX) because we need to generate images without them appearing on the screen (e.g. for report generation).
I will try to think about it.

BTW, the version 3.0.2.10 will be able to display bottle packs with trays which can be modified to look like those opened crates that you need.
![image](https://user-images.githubusercontent.com/1512039/78186442-0d342580-746d-11ea-86ee-e144d8e55cc3.png)

**Comment by AndrewF74** [comment]
Yeah, I knew texture to curved surfaces would not be in any current projection. Too much calculus for me… 😊

Very Nice with the modified tray and Thank you… 😊

Andrew D. Forsythe
Receiving Manager
andrewf@4life.com<mailto:andrewf@4life.com>
Phone:801.562.3729
Receiving Group: 801. 256.3007 or x 23007
Fax: 801.562.3635
Biomedical Research Laboratories LLC
4Life Research LLC
"Science, Service, & Success"
